src/Handler/Setup.php

```diff
@@ -44,12 +45,16 @@ public function __construct(
         $this->options = $this->normalizeOptions($options);
     }
 
-    public function handle(callable $notifier)
+    public function handle(callable $notifier, callable $existingFileCallback = null)
     {
         $env = EnvFactory::createEnv($this->envName, $this->appName, $this->dependencies);
 
-        foreach (Dumper::dump($env, $this->cwd, $this->getDumperFlags()) as $target) {
-            $notifier(str_replace($this->cwd.'/', '', $target));
+        try {
+            foreach (Dumper::dump($env, $this->cwd, $this->getDumperFlags(), $existingFileCallback) as $target) {
+                $notifier(str_replace($this->cwd.'/', '', $target));
+            }
+        } catch (\RuntimeException $e) {
```

Comment:
I did this so both `FailedDumpException` and (filesystem) `IOException` are caught